src/discussions/data/hooks.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable import/prefer-default-export */
+import { useContext, useEffect } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory, useLocation, useRouteMatch } from 'react-router';
+
+import { AppContext } from '@edx/frontend-platform/react';
+import { breakpoints, useWindowSize } from '@edx/paragon';
+
+import { Routes } from '../../data/constants';
+import { fetchCourseBlocks } from '../../data/thunks';
+import { clearRedirect } from '../posts/data';
+import { fetchCourseTopics } from '../topics/data/thunks';
+import { discussionsPath } from '../utils';
+import { selectAreThreadsFiltered } from './selectors';
+import { fetchCourseConfig } from './thunks';
+
+export function usePostThreadCount() {
+  return useSelector((state) => state.threads.totalThreads);
+}
```

Comment:
I don't really see the need fof this hook since it's just working as a selector. Instead create a selector and use that with useSelector.

Reply:
Updated.